vscode/package.json

```diff
@@ -201,6 +207,13 @@
           "description": "Whether analysis of file should be run when saved",
           "scope": "window",
           "order": 9
+        },
+        "konveyor.enableGenerativeAI": {
```

Comment:
Don't forget the kai/rpc-server path, these two configuration options should be near the top with the analyzer binary.